src/main/java/it/eng/idsa/businesslogic/processor/sender/SenderParseReceivedDataProcessorBodyFormData.java

```diff
@@ -48,7 +52,7 @@ public void process(Exchange exchange) throws Exception {
 		// Get from the input "exchange"
 		Map<String, Object> receivedDataHeader = exchange.getMessage().getHeaders();
 		logger.info("Received multipart/form request");
-
+		MultipartMessage multipartMessage = null;
```

Comment:
Can we put this declaration back where it is needed?
Check also other SenderParseReceivedData processors